frontend/src/bundles/common/components/protected-route/protected-route.tsx

```diff
@@ -21,7 +21,7 @@ const ProtectedRoute: React.FC<Properties> = ({ children }) => {
         );
     }
 
-    if (!user) {
+    if (dataStatus === DataStatus.REJECTED) {
```

Comment:
we change dataStatus in many cases not only in fetching the user, so if some async operation fails the user will be redirected to sign-in

Reply:
There are only 3 cases where we change dataStatus in auth slice, all three if failed should get or maintain the user to auth page. Also if I use `if (!user)` again, we'll have the same problem not accessing studio page again.

Reply:
Sorry if it sounded rude, it wasn't my intention at all!